language/phpmailer.lang-ja.php

```diff
@@ -10,19 +10,19 @@
 $PHPMAILER_LANG['authenticate']         = 'SMTPエラー: 認証できませんでした。';
 $PHPMAILER_LANG['connect_host']         = 'SMTPエラー: SMTPホストに接続できませんでした。';
 $PHPMAILER_LANG['data_not_accepted']    = 'SMTPエラー: データが受け付けられませんでした。';
-//$PHPMAILER_LANG['empty_message']        = 'Message body empty';
+$PHPMAILER_LANG['empty_message']        = 'メール本文が空です。';
 $PHPMAILER_LANG['encoding']             = '不明なエンコーディング: ';
 $PHPMAILER_LANG['execute']              = '実行できませんでした: ';
 $PHPMAILER_LANG['file_access']          = 'ファイルにアクセスできません: ';
 $PHPMAILER_LANG['file_open']            = 'ファイルエラー: ファイルを開けません: ';
 $PHPMAILER_LANG['from_failed']          = 'Fromアドレスを登録する際にエラーが発生しました: ';
 $PHPMAILER_LANG['instantiate']          = 'メール関数が正常に動作しませんでした。';
-//$PHPMAILER_LANG['invalid_address']      = 'Invalid address: ';
+$PHPMAILER_LANG['invalid_address']      = '不正なメールアドレス: ';
 $PHPMAILER_LANG['provide_address']      = '少なくとも1つメールアドレスを 指定する必要があります。';
-$PHPMAILER_LANG['mailer_not_supported'] = ' メーラーがサポートされていません。';
+$PHPMAILER_LANG['mailer_not_supported'] = 'メーラーがサポートされていません。';
```

Comment:
This leading space is needed, so please put it back. When its\ used, this string is immediately preceded by the name of the mailer that is not supported.